GOVERNANCE.md

```diff
@@ -71,7 +71,13 @@ The Buildpack Authors' Tooling team is responsible for helping buildpack authors
 ## Emeritus Team
 Serving as a member of an open source project requires a huge amount of work that cannot be sustained indefinitely. The emeritus team includes members of the project to whom we will always be grateful, but who no longer actively participate in the project.
 
-Project members will graduate to the emeritus team with a supermajority vote from the core team, and after giving notice that they no longer intend to consistently participate in the project.
+Project members may graduate to the emeritus team through one of the following methods:
+
+* a supermajority vote from the core team
```

Comment:
Note: As previously written, it read as though a supermajority vote was always necessary. This change makes it only one of the independent methods of graduating to the emeritus team.